invenio_app_ils/documents/mappings/os-v1/documents/document-v2.0.0.json

```diff
@@ -0,0 +1,978 @@
+{
+  "settings": {
+    "analysis": {
+      "normalizer": {
+        "custom_normalizer": {
+          "type": "custom",
+          "char_filter": [
+            "strip_special_chars"
+          ],
+          "filter": [
+            "lowercase",
+            "asciifolding"
+          ]
+        }
+      },
+      "analyzer": {
+        "min_length_2": {
+          "tokenizer": "standard",
+          "filter": [ "min_length_2" ]
+        },
+        "custom_analyzer": {
```

Comment:
correct me if I am wrong, but if we use this analyzer as search_analyzer per field, ie. for `title` it will work only if the search query will be `title:l'occasion` not `l'occasion`. "Global" query string `l'occasion` will still fail on the apostrophe, right ?

Reply:
We are using this custom_analyser as an `index_analyzer` too,  so it will work for both, lucene syntax and global.